sysutils/lscpu/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                lscpu
+version             1.2.0
+revision            0
+github.setup        NanXiao ${name} ${version}
```

Comment:
You can get rid of the `name` and `revision` fields, and use only `github.setup`:

```
github.setup        NanXiao lspcu 1.2.0
revision            0
```

`name` and `version` are defined by `github.setup` when done this way.

Reply:
I think you mean "get rid of name and version", not "name and revision".

Reply:
Correct, updated